Scripts/uetorch.lua

```diff
@@ -508,6 +510,7 @@ uetorch.SetActorVisible = utlib.SetActorVisible
 uetorch.SetActorVelocity = utlib.SetActorVelocity
 uetorch.SetActorAngularVelocity = utlib.SetActorAngularVelocity
 uetorch.SetActorScale3D = utlib.SetActorScale3D
+uetorch.DestroyActor = utlib.DestroyActor
```

Comment:
We are doing an offline step to detect if an actor is not visible at the moment, and for that we remove the light and the font, like this: https://github.com/bootphon/NaivePhysics/blob/master/Scripts/naive_physics.lua#L124